helm/templates/deployment.yaml

```diff
@@ -21,13 +21,19 @@ spec:
     metadata:
       labels:
         release: {{ .Release.Name }}
-      {{- with .Values.podLabels }}
+        {{- with .Values.podLabels }}
         {{- toYaml . | nindent 8 }}
-      {{- end }}
+        {{- end }}
+        {{- with .Values.commonPodLabels }}
```

Comment:
Do we need to go add this for every chart now too?

Reply:
I am adding them. this is a requirement from Fidelity to have some mandatory labels.